DataProcessing/datax-host/pom.xml

```diff
@@ -125,6 +125,11 @@ SOFTWARE
             <artifactId>datax-utility_2.4_2.11</artifactId>
             <version>1.2.3-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>com.microsoft.datax</groupId>
+            <artifactId>azure-keyvault-cryptography</artifactId>
```

Comment:
Is this needed if below exists

Reply:
yes this is needed in each respective package to be able to build each package separately